smac/facade/smac_ac_facade.py

```diff
@@ -436,11 +439,8 @@ def __init__(self,
         if intensifier_kwargs is not None:
             intensifier_def_kwargs.update(intensifier_kwargs)
 
-        if intensifier is None:
-            intensifier_instance = (
-                Intensifier(**intensifier_def_kwargs)  # type: ignore[arg-type] # noqa F821
-            )  # type: AbstractRacer
-        elif inspect.isclass(intensifier):
+        assert intensifier is not None
+        if inspect.isclass(intensifier):
```

Comment:
here we check if intensifier is a class but in line 433 we can have an instance. In that case the error would be raised. We should add a check if intensifier is an instance and if so, adopt it.

Reply:
Done.